src/main/java/org/sonarlint/intellij/config/global/wizard/NotificationsStep.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2020 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.config.global.wizard;
+
+import com.intellij.ide.wizard.AbstractWizardStepEx;
+import com.intellij.ui.BrowserHyperlinkListener;
+import com.intellij.util.ui.SwingHelper;
+import javax.swing.JCheckBox;
+import javax.swing.JComponent;
+import javax.swing.JEditorPane;
+import javax.swing.JPanel;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class NotificationsStep extends AbstractWizardStepEx {
+  private final WizardModel model;
+  private final boolean onlyEditNotifications;
+  private JPanel panel;
+  private JCheckBox notificationsCheckBox;
+  private JEditorPane notificationsDetails;
+
+  public NotificationsStep(WizardModel model, boolean onlyEditNotifications) {
+    super("Configure Notifications");
+    this.model = model;
+    this.onlyEditNotifications = onlyEditNotifications;
+  }
+
+  @Override
+  public JComponent getComponent() {
+    return panel;
+  }
+
+  @Override
+  public void _init() {
+    final boolean isSc = model.getServerType() == WizardModel.ServerType.SONARCLOUD;
+    final String sqOrSc = isSc ? "SonarCloud" : "SonarQube";
+    notificationsCheckBox.setText("Receive notifications from " + sqOrSc);
+    notificationsCheckBox.setVisible(model.isNotificationsSupported());
+    notificationsCheckBox.setEnabled(model.isNotificationsSupported());
```

Comment:
It was already there before but if the check box is not visible I guess we don't need to disable it. So this line could be removed

Reply:
In fact, now the entire wizard step is skipped if notifications are not supported, so all this setEnabled/setVisible studd is useless.